stacker/config/__init__.py

```diff
@@ -45,6 +45,16 @@ def render_parse_load(raw_config, environment=None, validate=True):
 
     rendered = render(raw_config, environment)
 
+    # Stage any remote package sources and merge into their defined
+    # configurations (if any)
+    pre_config = yaml.safe_load(rendered)
```

Comment:
This feels like it should be it's own method that gets called here, so we can keep `render_parse_load` clean.  It might even be better to be called either in the rendered or the parse steps themselves.